### PR TITLE
test for errorPopover lwc component

### PR DESCRIPTION
**Pull request by msrivastav13**
### What does this PR do?
test for errorPopover lwc component
### What issues does this PR fix or reference?

#<Insert GitHub Issue>

## The PR fulfills these requirements:

[ ] Tests for the proposed changes have been added/updated.
[ ] Code linting and formatting was performed.

### Functionality Before

<insert gif and/or summary>

### Functionality After

<insert gif and/or summary>

### CODE REVIEW

File: force-app/main/default/lwc/errorPopover/__tests__/errorPopover.test.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+describe('c-error-popover', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('renders c-error-popover element', () => {
```
Comment (pozil) [inline force-app/main/default/lwc/errorPopover/__tests__/errorPopover.test.js:12]: Don't test the static part of the markup. Only test what's dynamic here.
In short, you can resume this test to just:
```js
const popoverDivEl = element.shadowRoot.querySelector(
    '.slds-var-popover__body'
);
expect(popoverDivEl.textContent).toBe(ERROR_MESSAGE_INPUT);
```

File: force-app/main/default/lwc/errorPopover/__tests__/errorPopover.test.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        const ERROR_MESSAGE_INPUT = 'Friendly Error Message';
+
+        const element = createElement('c-error-popover', {
+            is: ErrorPopover
+        });
+        element.errors = ERROR_MESSAGE_INPUT;
+        document.body.appendChild(element);
+
+        const sectionEl = element.shadowRoot.querySelector('section');
+        expect(sectionEl).not.toBeNull();
```
[the remark this answers is not in the document]
Comment (msrivastav13) [reply force-app/main/default/lwc/errorPopover/__tests__/errorPopover.test.js]: done!

File: force-app/main/default/lwc/errorPopover/__tests__/errorPopover.test.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        expect(headerTitleEl.textContent).toBe('Resolve error');
+
+        //check for errors in final div
+        const popoverDivEl = element.shadowRoot.querySelector(
+            'div.slds-var-popover__body'
+        );
+        expect(popoverDivEl.textContent).toBe(ERROR_MESSAGE_INPUT);
+    });
+
+    it('clicks close Dialog buton', () => {
```
[the remark this answers is not in the document]
Comment (msrivastav13) [reply force-app/main/default/lwc/errorPopover/__tests__/errorPopover.test.js]: done

File: force-app/main/default/lwc/errorPopover/__tests__/errorPopover.test.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+        const ERROR_MESSAGE_INPUT = 'Friendly Error Message';
+
+        const element = createElement('c-error-popover', {
+            is: ErrorPopover
+        });
+        element.errors = ERROR_MESSAGE_INPUT;
+        document.body.appendChild(element);
+
+        const sectionEl = element.shadowRoot.querySelector('section');
+        expect(sectionEl).not.toBeNull();
```
[the remark this answers is not in the document]
Comment (msrivastav13) [reply force-app/main/default/lwc/errorPopover/__tests__/errorPopover.test.js]: done

File: force-app/main/default/lwc/errorPopover/__tests__/errorPopover.test.js (hunk 1 above)
Comment (msrivastav13) [reply force-app/main/default/lwc/errorPopover/__tests__/errorPopover.test.js:12]: done

**Pull request merged**